src/main/java/org/cloudfoundry/promregator/endpoint/AbstractMetricsEndpoint.java

```diff
@@ -86,7 +86,7 @@
 
 	@Value("${promregator.endpoint.maxProcessingTime:#{null}}")
 	@Deprecated
-	/**
+	/*
 	 * use maxProcessingTime instead
```

Comment:
```suggestion
	 * The maximal processing time permitted for Scraping (in milliseconds). The value is deprecated as it originates from the deprecated configuration option <pre>promregator.endpoint.maxProcessingTime</pre>. Use maxProcessingTime instead.
```

Reply:
So I worked out what a dangling Javadoc is now

Its where the javadoc is not completely above the property/method and its annotations.

Apparently, it needs to be before any annotations..... updated...